### DISCUSSION TITLE
is this possible?

### DISCUSSION THREAD

**Discussion by nikhil-varma-mudunuri** [pocketbase/pocketbase#4052] [Q&A] [answered]
here is my scenario 
collection A: player_id, score
collection B: userId,listOfPlayer_ids (player_id is a relation to collection A)
i need a view: userId, sum (listOfPlayer_ids.score)

**Comment by ganigeorgiev** [comment, marked as answer]
If `listOfPlayer_ids` is a multiple `relation` field, then the value will be stored as serialized json array and your view query could look something like:

```sql
SELECT
    b.user_id as id,
    b.user_id,
    SUM(a.score) as totalScore
FROM collectionB as b
LEFT JOIN json_each(b.listOfPlayer_ids) as lp
LEFT JOIN collectionA as a ON a.id = lp.value
GROUP BY b.user_id
```
_(for all available SQLite json function check https://www.sqlite.org/json1.html)_
